plugins/PrepareStage/PrepareMenu.qml

```diff
@@ -46,16 +46,16 @@ Item
         Cura.MachineSelector
         {
             id: machineSelection
-            width: UM.Theme.getSize("sidebar").width
+            width: Math.round(0.8 * UM.Theme.getSize("sidebar").width) - configSelection.width
```

Comment:
Should we not just adjust the theme to be 80% of what it used to be?

Reply:
I just did this to fit the minimum window width, but of course it will change in the restyling of the stage bar. I will keep it like this for now until there is a final decision on the sizes.

Reply:
I created a new entry for the size of the machine selector in 90e8a05aab63d5d16a2431571f6e9b6791ccd192
Now it's called `machine_selector_widget`